cmd/postcli/main.go

```diff
@@ -167,10 +167,7 @@ func main() {
 	}
 
 	if verifyPos {
-		if cmdVerifyPos(opts, fraction, logger) != nil {
-			os.Exit(1)
-		}
-		os.Exit(0)
+		cmdVerifyPos(opts, fraction, logger)
```

Comment:
Previously it was clear that the program terminates after calling `cmdVerifyPos()`. Now it's hidden and less obvious. How about returning an `error` and adding a `log.Fatal()` here?

Reply:
I don't see the benefit? Errors are not logging messages. I changed the code so it is clear it exits after `cmdVerifyPos` is called.

Reply:
> Errors are not logging messages.

? The point of CLI is to communicate to the user the outcome. What's wrong with communicating by showing them a nice readable error message?

>I don't see the benefit?

The benefit is that the next person reading this code immediately understands that the program terminates here.

Reply:
> What's wrong with communicating by showing them a nice readable error message?

I did this by updating the errors with your suggestions and printing them directly? Why first wrap them in an error to return them and then print them?

> The benefit is that the next person reading this code immediately understands that the program terminates here.

That's why I added the `os.Exit(0)` as you suggested?